src/main/kotlin/nevam/AppModule.kt

```diff
@@ -25,4 +25,14 @@ class AppModule(user: NexusUser, debugMode: Boolean, val poms: List<Pom>) {
       config = NexusConfig.DEFAULT,
       scheduler = Schedulers.single()
   )
+
+  private companion object {
+    private fun repositoryUrl(hostPrefix: String): String {
+      val actualPrefix = if (hostPrefix.isEmpty())
+        ""
+      else
+        "$hostPrefix."
```

Comment:
nit: Braces would be nice in if-else conditions. I've been burned by these so many times.